src/client/src/services/limitChecker/limitChecker.openfin.ts

```diff
@@ -1,128 +1,56 @@
-/* eslint-disable @typescript-eslint/ban-ts-comment */
-// This file uses legacy OpenFin API which is not compatible with the
-// types of the version of the OpenFin lib we are using. This should
-// be fixed, but in the meantime, we are ignoring type checking on
-// on this file.
-// @ts-nocheck
 import { bind } from "@react-rxjs/core"
-import { BehaviorSubject, of, race, Subject, timer } from "rxjs"
-import { map, switchMap, take, tap } from "rxjs/operators"
+import { BehaviorSubject, race, Subject, timer } from "rxjs"
+import { map, take, tap } from "rxjs/operators"
 
 import { checkLimitFn } from "./types"
 
-const limitCheckSubscriptionUuid$ = new BehaviorSubject<string | null>(null)
+window.fdc3.addContextListener("limit-checker-status", (context) => {
+  if (context.id) limitCheckSubscription$.next(context.id.isAlive)
```

Comment:
nit: if you are making other changes .. really not a fan of inline if statements 😞